### DISCUSSION TITLE
Request a HTML version

### DISCUSSION THREAD

**Discussion by jingxixu** [alshedivat/al-folio#228] [Q&A] [answered]
Hi,

I really like the design and look of this theme. However, I am more used to writing HTML directly but now if I take the generated `_site` on its own, the website won't look the same as using jekyll. I am wondering if a version of pure html + css can be provided? Many thanks!

**Comment by alshedivat** [comment]
@jingxixu, thanks for bringing this up. Could you explain what exactly you mean by `the website won't look the same as using jekyll`? Jekyll essentially displays the static website generated in the `_site` folder. It may not look the same b/c the URLs to resources (styles and scripts) aren't set correctly after you copy the HTML version. Jekyll takes care of setting all relative paths correctly at generation time.

**Comment by pdeperth** [comment]
Hello! 
I have the same problem: after running the static site generation with `bundle exec jekyll build --lsi` and copying the contents of the `_site` directory to my server, the displayed result on the web page looks very ugly. Thank you for your help.

**Reply by pdeperth** [reply]
@alshedivat were you saying something else needs to be done?

**Reply by pdeperth** [reply]
Hello! 
Does anyone have a clue about this?

**Reply by pdeperth** [reply]
@jingxixu did you find out?

**Reply by alshedivat** [reply]
you'll have to inspect if the paths to css and js files were set correctly throughout all the generated files in the `_site` folder. make sure you set `url` and `baseurl` in `_config.yml` correctly when building the website. (technically, we don't support manual copying of the files from `_site`, so you are a bit on your own here.)

**Reply by pdeperth** [reply]
Hello @alshedivat, thank you for answering me. 
Is, the `url` and `baseurl` are correct; about the paths to ccs and js files, do you know where I can check this in the folder?
Best,

**Comment by george-gca** [comment, marked as answer]
In your case you should first run your site locally straight from `_site/` directory. Try this:
1 - install some http server, for example with node `npm install -g http-server` or you can use the one that comes with python3
2 - run the http server pointing to `_site/`:
  - with node: `npx http-server _site/`
  - with python3: `python3 -m http.server -d _site/`

3 - open the local link in your browser, something like `localhost:8000` or another port depending on what your terminal says

If the site is not displayed correctly you set something wrong. Probably it is `baseurl`, which you should leave empty for this case.

**Comment by george-gca** [comment]
Also our template uses html/javascript/css underneath. You can have a look at [CUSTOMIZE.md](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md) to get an overview of the template structure. Basically the pages, like [about.md](https://github.com/alshedivat/al-folio/blob/master/_pages/about.md), are based on some html/liquid template defined in the page frontmatter, which for `about.md` is `layout: about`, located in [_layouts/about.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/about.liquid).
